.travis.yml

```diff
@@ -81,6 +81,12 @@ install:
   #- conda install -y scikit-beam -c nsls2forge
   #- conda update -y scikit-beam -c nsls2forge  # Update, because sometimes a very old version is installed :(
   - python -m pip install --upgrade pip setuptools
+  - |
+    set -e
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      # There are issues in building wheels for pystackreg>0.2.2 on OSX, so install pystackreg from CF
+      conda install -y pystackreg -c conda-forge
```

Comment:
I would propose using it from `nsls2forge` (https://github.com/nsls-ii-forge/pystackreg-feedstock has all successful builds for Python 3.7-3.9 for Linux/OSX/Windows), but given we are going to move to conda-forge anyway, let's keep it here.

Reply:
I think the main problem with nsls2forge is that `pyqt` from `defaults` channel is way out of date. I think this is one of the reasons I switched to conda-forge some time in the fall. It was also discussed that we are moving to `conda-forge`.